app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -149,6 +151,112 @@ class ExplorationActivityTest {
     explorationDataController.stopPlayingExploration()
   }
 
+  @Test
+  fun testExploration_overflowMenu_isDisplayedSuccessfully() {
+    getApplicationDependencies(TEST_EXPLORATION_ID_2)
+    launch<ExplorationActivity>(
+      createExplorationActivityIntent(
+        internalProfileId,
+        TEST_TOPIC_ID_0,
+        TEST_STORY_ID_0,
+        TEST_EXPLORATION_ID_2
+      )
+    ).use {
+      openActionBarOverflowOrOptionsMenu(context)
+      onView(withText(context.getString(R.string.menu_options))).check(matches(isDisplayed()))
+      onView(withText(context.getString(R.string.help))).check(matches(isDisplayed()))
+    }
+    explorationDataController.stopPlayingExploration()
+  }
+
+  @Test
+  fun testExploration_openOverflowMenu_selectHelpInOverflowMenu_showsHelpFragmentSuccessfully() {
+    getApplicationDependencies(TEST_EXPLORATION_ID_2)
+    launch<ExplorationActivity>(
+      createExplorationActivityIntent(
+        internalProfileId,
+        TEST_TOPIC_ID_0,
+        TEST_STORY_ID_0,
+        TEST_EXPLORATION_ID_2
+      )
+    ).use {
+      openActionBarOverflowOrOptionsMenu(context)
+      onView(withText(context.getString(R.string.help))).perform(click())
+      onView(
+        Matchers.allOf(
+          Matchers.instanceOf(TextView::class.java),
+          withParent(withId(R.id.help_activity_toolbar))
+        )
+      ).check(matches(withText(R.string.help)))
+    }
+    explorationDataController.stopPlayingExploration()
+  }
+
+  @Test
+  fun testExploration_openOverflowMenu_selectOptionsInOverflowMenu_showsOptionsFragmentSuccessfully() { // ktlint-disable max-line-length
+    getApplicationDependencies(TEST_EXPLORATION_ID_2)
+    launch<ExplorationActivity>(
+      createExplorationActivityIntent(
+        internalProfileId,
+        TEST_TOPIC_ID_0,
+        TEST_STORY_ID_0,
+        TEST_EXPLORATION_ID_2
+      )
+    ).use {
+      openActionBarOverflowOrOptionsMenu(context)
+      onView(withText(context.getString(R.string.menu_options))).perform(click())
+      onView(
+        Matchers.allOf(
+          Matchers.instanceOf(TextView::class.java),
+          withParent(withId(R.id.options_activity_toolbar))
+        )
+      ).check(matches(withText(R.string.menu_options)))
+    }
+    explorationDataController.stopPlayingExploration()
+  }
+
+  @Test
+  fun testExploration_openOverflowMenu_selectHelpInOverflowMenu_clickBackArrow_backToExplorationActivitySuccessfully() { // ktlint-disable max-line-length
```

Comment:
Remove this test case, Units test should not have steps across two different activities.

At best we can only check if the passed intent is correct or not. And rest should be check in the test file of the other activity.

Reply:
Done